adam-core/src/test/resources/sorted-variants.vcf

```diff
@@ -0,0 +1,74 @@
+##fileformat=VCFv4.2
```

Comment:
`sorted-variants.vcf` is the site-only `-only_variants` version for `TransformVariants`